cosmo_tester/test_suites/test_blueprints/nodecellar_test_autoheal.py

```diff
@@ -139,14 +130,20 @@ def post_autoheal_assertions(self, before_state, after_state):
 
     def modify_blueprint(self):
         with YamlPatcher(self.blueprint_yaml) as patch:
-            patch.merge_obj('workflows', self.AUTOHEAL_WORKFLOW_YAML)
             patch.merge_obj('groups', self.AUTOHEAL_GROUP_YAML)
-            # FIXME: this is a workaround for multiple collectors starting
-            # multiple auto-heal workflows
-            patch.set_value("node_types.nodecellar\\.nodes\\.MonitoredServer."
+            patch.merge_obj("node_types.nodecellar\\.nodes\\.MonitoredServer."
                             'interfaces.cloudify\\.interfaces\\.monitoring.'
                             'start.inputs.collectors_config.default',
                             {'ExampleCollector': {}})
+            # TODO: Remove this link replacement when suitable
+            imports = patch._get_object_by_path('imports')
+            imports = [
+                imp if 'types.yaml' not in imp
+                else 'https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/master'
+                     '/resources/rest-service/cloudify/types/types.yaml'
+                for imp in imports
+            ]
+            patch.set_value('imports', imports)
```

Comment:
Yes, I have just compared it with http://www.getcloudify.org/spec/cloudify/3.2m3/types.yaml and it is identical. I will remove this TODO, it doesn't pass flake8 anyway